tests/SprykerEcoTest/Zed/Unzer/_support/UnzerZedTester.php

```diff
@@ -0,0 +1,565 @@
+
+    /**
+     * @var string
+     */
+    public const ADDRESS_COUNTRY = 'Germany';
+
+    /**
+     * @var string
+     */
+    public const ADDRESS_CITY = 'Berlin';
+
+    /**
+     * @var string
+     */
+    public const ADDRESS_ZIP = '20537';
+
+    /**
+     * @var string
+     */
+    public const ADDRESS_ADDITIONAL = 'street';
+
+    /**
+     * @var int
+     */
+    public const ADDRESS_NO = 324324;
+
+    /**
+     * @var string
+     */
+    public const ADDRESS_STREET = 'Street';
+
+    /**
+     * @var string
+     */
+    public const CUSTOMER_SALUTATION = 'Mr';
+
+    /**
+     * @var string
+     */
+    public const CUSTOMER_SURNAME = 'Doe';
+
+    /**
+     * @var string
+     */
+    public const CUSTOMER_NAME = 'John';
+
+    /**
+     * @var string
+     */
+    public const CUSTOMER_EMAIL = 'john.doe@mail.com';
+
+    /**
+     * @var string
+     */
+    public const CUSTOMER_REFERENCE = 'DE-22';
+
+    /**
+     * @var string
+     */
+    public const PAYMENT_METHOD = 'unzerMarketplaceBankTransfer';
+
+    /**
+     * @var string
+     */
+    public const PAYMENT_PROVIDER = 'unzer';
+
+    /**
+     * @var string
+     */
+    public const UNZER_API_RESPONSE_CUSTOMER_ID = 's-cust-43434234';
+
+    /**
+     * @var string
+     */
+    public const UNZER_API_RESPONSE_METADATA_ID = 's-meta-5423423';
+
+    /**
+     * @var string
+     */
+    public const UNZER_API_RESPONSE_BASKET_ID = 's-bskt-435345';
+
+    /**
+     * @var string
+     */
+    public const UNZER_API_RESPONSE_PAYMENT_RESOURCE_ID = 's-type-sofort-1';
+
+    /**
+     * @var string
+     */
+    public const UNZER_API_RESPONSE_WEBHOOK_URL = 'https://unzer-spryker.com';
+
+    /**
+     * @var string
+     */
+    public const UNZER_KEYPAIR_ID = 'key-1';
+
+    public const UNZER_MAIN_REGULAR_KEYPAIR_ID = 'keypair-id-1';
+    public const UNZER_MAIN_MARKETPLACE_KEYPAIR_ID = 'keypair-id-2';
+    public const UNZER_MARKETPLACE_MAIN_MERCHANT_KEYPAIR_ID = 'keypair-id-3';
+    public const UNZER_MARKETPLACE_MERCHANT_KEYPAIR_ID = 'keypair-id-4';
+
+    public const MERCHANT_REFERENCE = 'merchant1';
+
+    /**
+     * @var string
+     */
+    public const UNZER_PUBLIC_KEY = 's-pub';
+
+    /**
+     * @var string
+     */
+    public const UNZER_PRIVATE_KEY = 's-priv';
+
+    /**
+     * @var string
+     */
+    public const UNZER_PAYMENT_ID = 's-pay-1234';
+
+    /**
+     * @var string
+     */
+    public const UNZER_EVENT_AUTHORIZED = 'authorize.succeeded';
+
+    /**
+     * @var string
+     */
+    public const ORDER_REFERENCE = 'DE--19';
+
+    /**
+     * @var string
+     */
+    public const STATE_MACHINE_PROCESS_NAME = 'UnzerMarketplaceBankTransfer01';
+
+    /**
+     * @param \Codeception\Scenario $scenario
+     */
+    public function __construct(Scenario $scenario)
+    {
+        parent::__construct($scenario);
+
+        $this->setUpConfig();
+    }
+
+    /**
+     * @return void
+     */
+    protected function setUpConfig(): void
```

Comment:
It's rather a matter of taste, but can we put `protected` methods below `public`?